docs/recipes/lhci-server-vpn-proxy/nginx.conf

```
@@ -0,0 +1,18 @@
+server {
+      listen      443;
+      server_name api-proxy.com;
```

```suggestion
      server_name public-proxy.example.com;
```

we try to stick to <subdomain>.example.com to ever prevent accidentally sending someone somewhere nefarious :)

maybe search and replace `api-proxy.com` with `public-proxy.example.com` and `vpn.lhci.com` with `lhc-over-vpn.example.com`?